src/format/Kdbx3XmlWriter.h

```diff
@@ -1,91 +0,0 @@
-/*
```

Comment:
If you wonder why this is in here: I snuck in deletion of this obsolete header file.